swap_meet/decor.py

```diff
@@ -1,2 +1,16 @@
-class Decor:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+import uuid
+
+class Decor(Item):  
+    def __init__(self, id=None, condition=None, age=None, width=None, length=None):
```

Comment:
You can use `width = 0` and `length = 0` as defaults, since ints don't have the same problem as `= []` in defaults.